regression/ansi-c/gcc_types_compatible_p1/main.c

```diff
@@ -67,6 +67,10 @@ STATIC_ASSERT(__builtin_types_compatible_p(typeof (hot) *, int *));
 STATIC_ASSERT(__builtin_types_compatible_p(typeof (hot), typeof (janette)));
 STATIC_ASSERT(__builtin_types_compatible_p(__int128, signed __int128));
 
+#if defined(__x86_64__) || defined(__i386__)
+STATIC_ASSERT(__builtin_types_compatible_p(__float128, _Float128));
+#endif
```

Comment:
This needs to be moved into the `#ifndef __clang__` block to get the regression test to pass on all platforms.